content/doc/tutorials/tutorial-for-installing-jenkins-on-IBM-Cloud.adoc

```diff
@@ -5,202 +5,225 @@ section: doc
 ---
 
 :toc:
-:toclevels: 3
+:toclevels: 4
 :imagesdir: ../../book/resources/
 
-You should have an IBM Cloud account, otherwise you can http://cloud.ibm.com/registration[register here].
+You should have an IBM Cloud account, otherwise you can link:http://cloud.ibm.com/registration[register here].
 At the end of the tutorial you will have a cluster with Jenkins up and running.
 
-1. We will provision a new Kubernetes Cluster for you if, you already have one skip to <<Step 2 deploy IBM Cloud Block Storage plug-in,step **2**>>
-2. We will deploy  the <<Step 2 deploy IBM Cloud Block Storage plug-in,IBM Cloud Block Storage plug-in>>, if already have it skip to <<Step 3 deploy Jenkins,step **3**>>
-3. <<Step 3 deploy Jenkins,Jenkins deployment>>
+. We will provision a new Kubernetes cluster for you, if you already have one skip to <<Deploy IBM Cloud Block Storage plug-in,step 2>>.
+. We will deploy  the <<Deploy IBM Cloud Block Storage plug-in,IBM Cloud Block Storage plug-in>>, if you already have it skip to <<Deploy Jenkins,step 3>>.
+. We will <<Deploy Jenkins,deploy Jenkins>>.
+. We will <<Verify Jenkins installation,verify jenkins installation>>.
+
+== Provision Kubernetes cluster
```

Comment:
I took a look at some of the other Jenkins' topics and many use gerunds for procedural steps, which also aligns with the CloudBees style guide. I think we could update these heading to align with the other topics.
```suggestion
== Provisioning the Kubernetes cluster
```